web/src/app/schedules/temp-sched/TempSchedAddShiftForm.tsx

```diff
@@ -54,14 +45,15 @@ export default function TempSchedAddShiftForm(): JSX.Element {
             label='Shift End'
             name='end'
             hint={
-              <Typography
-                className={classes.typography}
-                variant='caption'
-                color='textSecondary'
-                onClick={() => setManualEntry(false)}
+              <AppLink
```

Comment:
AppLink isn't for styling, it's meant for link/nav/tab things, could render it as an `<a>` tag if we want it as a "link" though (that's all AppLink is doing visually).

Reply:
Based on the link from the linter, we should use `<button>` and add the appropriate styles for accessibility. It looks like they have them a little lower on the page that can be copy/pasted for the most part.

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md